backend/dissemination/workbooklib/end_to_end_core.py

```diff
@@ -226,7 +245,14 @@ def generate_workbooks(user, email, dbkey, year, store_files=True, run_api_check
 def run_end_to_end(email, dbkey, year, store_files=True, run_api_checks=True):
     try:
         user = User.objects.get(email=email)
+
     except User.DoesNotExist:
-        logger.info("No user found for %s, have you logged in once?", email)
-        return
+        # logger.info("No user found for %s, have you logged in once?", email)
+        logger.info("Creating a user for test data generation.")
+        test_user_email = "test-data-generator@fac.gsa.gov"
+        try:
+            user = User.objects.get(email=test_user_email)
+        except Exception:
+            user = User.objects.create(email=test_user_email)
+        # return
```

Comment:
```suggestion
        test_user_email = "test-data-generator@fac.gsa.gov"
        try:
            user = User.objects.get(email=test_user_email)
        except Exception:
            logger.info("Creating a user for test data generation.")
            user = User.objects.create(email=test_user_email)

```

Reply:
suggest using `get_or_create` here to collapse this into a single statement: https://docs.djangoproject.com/en/5.0/ref/models/querysets/#get-or-create